src/Cron/MinutelyCron.php

```diff
@@ -12,13 +12,13 @@
  * @author     Moritz Vondano
  */
 
-namespace Mvo\ContaoFacebookImport\EventListener;
+namespace Mvo\ContaoFacebookImport\Cron;
 
 use Mvo\ContaoFacebookImport\Synchronization\Scheduler;
 use Symfony\Component\DependencyInjection\ContainerAwareInterface;
 use Symfony\Component\DependencyInjection\ContainerAwareTrait;
 
-class ContaoCronListener implements ContainerAwareInterface
+class MinutelyCron implements ContainerAwareInterface
```

Comment:
What's the reason for the class + method rename? Before you did know what the class was doing, not so now. :stuck_out_tongue:

Reply:
Also copy & paste: https://docs.contao.org/dev/framework/cron/#scope and it's in the `Cron` scope, so you should know what it's doing, don't you?

Reply:
I tend to disagree. It's the same with naming event listener methods instead of what they are actually doing (e.g. compare `onClickEvent()` vs. `login()`).

Reply:
I reverted the renaming of ContaoCronListener  ;-)